docs/VideosApi.md

```diff
@@ -615,8 +621,9 @@ Name | Type | Description  | Notes
 ### HTTP response details
 | Status code | Description | Response headers |
 |-------------|-------------|------------------|
-**200** | Success |  -  |
+**200** | Success |  * X-RateLimit-Limit - The request limit per minute. <br>  * X-RateLimit-Remaining - The number of available requests left for the current time window. <br>  * X-RateLimit-RetryAfter - The number of seconds left until the current rate limit window resets. <br>  |
```

Comment:
@szekelyzol formating is strange here

Reply:
I think this was autogenerated 😢

Reply:
Well it is properly displayed there: https://github.com/apivideo/api.video-java-client/blob/api-rate-limiting-oas/docs/VideosApi.md#http-response-details-6